src/bls12_381/ec.rs

```diff
@@ -670,15 +670,15 @@ macro_rules! curve_impl {
                 let mut hasher_1 = hasher_0.clone();
 
                 hasher_0.update(b"_0");
```

Comment:
Couldn't the hasher's also share the "_" before the final 0 and 1 bytes are appended? (Or would that not actually reduce computation time?)

Reply:
Haha, yeah. It would help if the _ sat next to a BLAKE2b block boundary. Certainly an implementation detail. :)